hand_control:
 grip_strength: 0.2